singleapplication.h

```diff
@@ -124,14 +124,24 @@ class SingleApplication : public QAPPLICATION_CLASS
      */
     QString currentUser() const;
 
+    /**
+     * @brief Mode of operation of sendMessage.
+     * @enum
+     */
+    enum SendMode {
+        NonBlocking,            /** Do not wait for the primary instance termination and return immediately */
```

Comment:
We usually do not alight such things:
```suggestion
        NonBlocking,  /** Do not wait for the primary instance termination and return immediately */
```

Reply:
Ok, sure, no problem with that, just thought as you aligned the asignments in the other mode flags enum that would match better.